### ISSUE TITLE
Dark Fog compatibility

### ISSUE THREAD

**Issue by PiggyWhiskey** [mattsemar/dsp-bulldozer#27]
Current DarkFog version experiences an error when selecting 'Bulldoze'

![image](https://github.com/mattsemar/dsp-bulldozer/assets/9373995/50a51004-792f-4db3-b4f5-39e670fd2e9e)

**Comment by WelshGuyX** [comment]
Yup can confirm this does not work with dark fog. Same error.. Would really appreciate a fix.

**Comment by JamieRCHI** [comment]
The new 'forked' version of this mod on Thunderstore does work in the latest DSP game with the Dark Fog.
Although the person that created the new version of the mod can't be contacted as far as I know.
It looks like this original version of the mod has been abandoned.

Edit: This version allows the game to start for me and it seems to work. 
But can't completely verify due to the 'Universe Exploration 3' research requirement.
Which is really stupid. Only the 'Universe Exploration 2' requirement should be needed
since this mod only deals directly with the Planet itself instead of the Entire solar system.
